prac_05/hex_colours.py

```diff
@@ -0,0 +1,18 @@
+"""
+CP1404Practical
+hexadecimal colour codes in a dictionary
+"""
+
+colour_to_code = {"Absolute Zero": "#0048ba", "Acid Green": "#b0bf1a", "AliceBlue": "#f0f8ff",
```

Comment:
This is a constant, therefore I think block capitals are the convention 😎

Reply:
Easy fix.